docs/basics/101-180-FAQ.rst

```diff
@@ -411,6 +411,21 @@ dataset.
 When using the terminal command ``cp`` [#f1]_, it is sufficient to use the
 ``-L``/``--dereference`` option. This will follow symbolic links, and make
 sure that content gets moved instead of symlinks.
+Remember that if you are copying some annexed content out of a dataset without
+unlocking it first, you will only have "read" access on the files you have just 
```

Comment:
There is a glossary entry on ``permissions`` in the book. How about this:
```suggestion
unlocking it first, you will only have "read" :term:`permissions` on the files you have just 
```

The glossary entry differentiates between read, write, and execute permissions, and maybe helps to understand the greater scheme of things

<details><summary>Here is what it says </summary>
permissions

 Access rights assigned by most file systems that determine whether a user can view (read permission), change (write permission), or execute (execute permission) a specific content.

-  read permissions grant the ability to a file, or the contents (file names) in a directory.

- write permissions grant the ability to modify a file. When content is stored in the object-tree by git-annex, your previously granted write permission for this content is revoked to prevent accidental modifications.

- execute permissions grant the ability to execute a file. Any script that should be an executable needs to get such permission.



</details>